docs/source/conf.py

```diff
@@ -0,0 +1,113 @@
+# Configuration file for the Sphinx documentation builder.
+#
+# For the full list of built-in configuration values, see the documentation:
+# https://www.sphinx-doc.org/en/master/usage/configuration.html
+
+# -- Project information -----------------------------------------------------
+# https://www.sphinx-doc.org/en/master/usage/configuration.html#project-information
+import os
+import sys
+
+sys.path.insert(0, os.path.abspath("../.."))
+
+project = "Service-Kit"
+copyright = "2025, Mike Salvatore"
+author = "Mike Salvatore"
+
+# The short X.Y version
+version = "0.0"
+# The full version, including alpha/beta/rc tags
+release = "0.0.0"
+
+extensions = [
+    "sphinx.ext.autodoc",  # support for automatic documentation of docstrings
+    "sphinx.ext.intersphinx",  # references to external documentation
+    "sphinx.ext.autosummary",
```

Comment:
@ilija-lazoroski Do we need this?

Reply:
I have removed it but not pushed. Now it is up there